dev/drivers/scripts/stats/cam/jevs_cam_hrrr_grid2obs_stats.sh

```diff
@@ -4,7 +4,8 @@
 #PBS -S /bin/bash
 #PBS -q dev
 #PBS -A VERF-DEV
-#PBS -l walltime=01:45:00
+#PBS -A VERF-DEV
```

Comment:
@PerryShafran-NOAA `#PBS -A VERF-DEV` seems to appear twice (on lines 6 and 7). Can line 7 be removed?

Reply:
Fixed.